### PR TITLE
Update media.html

### PR DESCRIPTION
**Pull request by dazole**
The video presentation is titled with "Puppet", but it is very Foreman-centric.  Hopefully that will be ok.

### CODE REVIEW

File: _layouts/media.html [hunk 1]
```diff
@@ -105,6 +105,7 @@
 
 var talks = [
 	{ id: "wSpPjUPIEVU", title: "OSDC 2015 Foreman in your data center", author: "Stephen Benjamin" },
+	{ id: "b18Us1nAKMA", title: "PuppetCamp 2015: Selling management and developers on Puppet", author: "David Patterson"},
```
Comment (stbenjam) [inline _layouts/media.html]: Is this the wrong ID? I think it should be `Qo3Bkm7dqlU`.

Comment (dazole) [inline _layouts/media.html]: Wow.  I don't even know how that happened.  Yes, you're correct, the ID should be Qo3Bkm7dqlU.

Being new at this, do I need to create a new pull request for that?  The embarrassment is palpable over here, heh.

Comment (dLobatog) [inline _layouts/media.html]: @Dan33l No need to create a new pull request, fix the mistake, `git commit --amend`, and --force push against the same branch. Thanks!

Comment (stbenjam) [inline _layouts/media.html]: No worries, I think if you're in a playlist YouTube doesn't update the top URL.  You can edit it in GitHub's web interface too (on the Files changed tab),  but @eLobato's amend suggestion is the best way to do it so it stays just one commit.

**Comment by dLobatog** [comment]
Merged as 930a17b6bf985c4d19f7cbe590ceac49b18e65fa, thanks @dazole !

**Pull request closed without merging**